### DISCUSSION TITLE
Cut method within "create cut" component doesn't work when cutting between plates

### DISCUSSION THREAD

**Discussion by jshort45** [idea-statica/ideastatica-public#370] [Rhino/Grasshopper] [answered]
Hi,
Using the create cut component, I find that when I cut a plate by another plate, it does not follow the "Cut Method" inputted. It seems to default to “Surface”, even when I explicitly say BoundingBox. See below images.

![IDEA-cut method1](https://github.com/idea-statica/ideastatica-public/assets/129746855/570ab871-d495-47a9-b1e5-fc295f863169)
![IDEA-cut method2](https://github.com/idea-statica/ideastatica-public/assets/129746855/2c6529a0-a9aa-485f-8d23-5efc79654bef)

**Comment by nathanlukeIDEA** [comment, marked as answer]
Hi, @jshort45 this has been fixed and will be released in IDEA StatiCa version 23.1.2. Nothing is required to be changed in the Grasshopper definition.
